pages/index.js

```diff
@@ -1,5 +1,12 @@
 import Link from "next/link";
 
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <>
+      <h1>Car App</h1>
+      <Link href="/profile/WAUZZZ8V9LA015917">Test Detail Page Car 1</Link>
+      <br></br>
```

Comment:
Please do not use html elements just for styling - use css instead - since it is not semantic.

Reply:
Ok thank you, I will delete them before the merge, because they were just there for testing reasons.